wuvt/templates/admin/donation_index.html

```diff
@@ -13,7 +13,7 @@ <h1>Donations</h1>
 
         <form method="post">
             <input type="hidden" name="_csrf_token" value="{{ csrf_token() }}">
-            <button name="reset_stats" type="submit" class="btn btn-danger btn-sm">
+            <button name="reset_stats" type="submit" class="btn btn-danger btn-sm" onclick="confirm_delete(event)">
```

Comment:
Would you mind adding this as an event listener with jQuery instead? There's already a static/js/admin/donation_index.js that you can use. Long term, I'd like to lock down the Content-Security-Policy for the app more (by removing `'unsafe-inline'`), which would break this.